src/Propel/Generator/Builder/Om/ObjectBuilder.php

```diff
@@ -6377,6 +6383,13 @@ protected function doSave(ConnectionInterface \$con" . ($reloadOnUpdate || $relo
                 }
 ";
             }
+
+            // Update primary key values cache
+            if($col->isPrimaryKey()) {
```

Comment:
also some CS issues here

Reply:
Thanks, will fix it (https://github.com/propelorm/Propel2/runs/7615260378?check_suite_focus=true#step:10:186)